<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заполнение областей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>811</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {display: inline-block;
                              font-size: 12px;
                              cursor: pointer;
                              text-align: center;
                              text-decoration: none;
                              outline: none;
                              color: #696969;
                              background-color: #fff;
                              border: none;
                              border-radius: 10px;
                              box-shadow: 0 9px #999;
                            }
QTableWidget:hover {background-color: #fff}
QTableWidget:pressed {
selection-background-color: #C0C0C0;
                              box-shadow: 0 5px #666;
                              transform: translateY(4px);
                            }</string>
    </property>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="paint">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>760</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                                display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                 box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
    </property>
    <property name="text">
     <string>Закрасить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>790</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>270</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Для ввода мышкой</string>
    </property>
    <widget class="QCheckBox" name="vertic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>вертикальная прямая</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="horiz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>горизонтальная прямая</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>370</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Для ввода с клавиатуры</string>
    </property>
    <widget class="QSpinBox" name="x">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
                              display: inline-block;
                              font-size: 16px;
                              cursor: pointer;
                              text-align: center;
                              text-decoration: none;
                              outline: none;
                              color: #696969;
                              background-color: #fff;
                              border: none;
                              border-radius: 10px;
                              box-shadow: 0 9px #999;
                            }
                            QSpinBox:hover {background-color: #fff}
                            QSpinBox:pressed {
                              selection-background-color: #C0C0C0;
                              box-shadow: 0 5px #666;
                              transform: translateY(4px);
                            }</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="y">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
                              display: inline-block;
                              font-size: 16px;
                              cursor: pointer;
                              text-align: center;
                              text-decoration: none;
                              outline: none;
                              color: #696969;
                              background-color: #fff;
                              border: none;
                              border-radius: 10px;
                              box-shadow: 0 9px #999;
                            }
                            QSpinBox:hover {background-color: #fff}
                            QSpinBox:pressed {
                              selection-background-color: #C0C0C0;
                              box-shadow: 0 5px #666;
                              transform: translateY(4px);
                            }</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                                display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                 box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
     </property>
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>470</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Затравочный пиксель</string>
    </property>
    <widget class="QSpinBox" name="p_x">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
                              display: inline-block;
                              font-size: 16px;
                              cursor: pointer;
                              text-align: center;
                              text-decoration: none;
                              outline: none;
                              color: #696969;
                              background-color: #fff;
                              border: none;
                              border-radius: 10px;
                              box-shadow: 0 9px #999;
                            }
                            QSpinBox:hover {background-color: #fff}
                            QSpinBox:pressed {
                              selection-background-color: #C0C0C0;
                              box-shadow: 0 5px #666;
                              transform: translateY(4px);
                            }</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="p_y">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
                              display: inline-block;
                              font-size: 16px;
                              cursor: pointer;
                              text-align: center;
                              text-decoration: none;
                              outline: none;
                              color: #696969;
                              background-color: #fff;
                              border: none;
                              border-radius: 10px;
                              box-shadow: 0 9px #999;
                            }
                            QSpinBox:hover {background-color: #fff}
                            QSpinBox:pressed {
                              selection-background-color: #C0C0C0;
                              box-shadow: 0 5px #666;
                              transform: translateY(4px);
                            }</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pixel_z">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                                display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                 box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
     </property>
     <property name="text">
      <string>Задать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>570</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Дополнительные функции</string>
    </property>
    <widget class="QCheckBox" name="delay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>С задержкой</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="timer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Засечь время</string>
     </property>
    </widget>
    <widget class="QPushButton" name="erase">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                                display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                 box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="lock">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>720</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                                display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                 box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                                }</string>
    </property>
    <property name="text">
     <string>Замкнуть</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>660</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Цвет закраски</string>
    </property>
    <widget class="QPushButton" name="addpoint_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 27, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 230, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(125, 111, 111);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(144, 250, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(219, 255, 135);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_13">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(180, 113, 160);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_14">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 85, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpoint_15">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(148, 33, 33);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
